src/Pages/Bookmarked.tsx

```diff
@@ -1,14 +1,33 @@
-import { Typography } from '@mui/material'
+import { Box, Typography } from '@mui/material'
 import moviesData from '../../data/movies.json'
 import ThumbnailCard from '../components/thumbnailCard/ThumbnailCard'
 
 function Bookmarked() {
   return (
-    <div>
+    <div
```

Comment:
Is it possible to switch this div for a Box from MUI instead? Maybe you should also move this styling to the styling-sheet.

Reply:
I fix!